src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -105,38 +97,46 @@ public void doInit() {
     rightFront.restoreFactoryDefaults();
     rightBack.restoreFactoryDefaults();
 
+    rightGroup.setInverted(true);
```

Comment:
maybe its just the diff... but it appears that you switched which side you decided was inverted (we had leftGroup.setInverted(true) on line 86 above and then here we have rightGroup.setInverted(true).

any reason for that?

Reply:
i was trying to match your PID code in the SparkMax Example. I believe there we are inverting right.